server/middleware/setUpWebSession.ts

```diff
@@ -25,7 +25,7 @@ export default function setUpWebSession(): Router {
       secret: config.session.secret,
       resave: false, // redis implements touch so shouldn't need this
       saveUninitialized: false,
-      rolling: true,
+      rolling: true, // This being true defeats the purpose of the below middleware.
```

Comment:
I'd noticed this before, but I think I found a reason for why it's done, though I now forget why that is. I would throw this on the #typescript channel to see what folks think of nuking it out of existence.